configs/multilabel_classification/mobilenetv3.yml

```diff
@@ -0,0 +1,89 @@
+lr_finder:
+  enable: True
+  mode: TPE
+  stop_after: True
+  num_epochs: 7
+  step: 0.001
+  epochs_warmup: 1
+  path_to_savefig: 'lr_finder.jpg'
+  max_lr: 1e-3
+  min_lr: 1e-5
+  n_trials: 15
+
+model:
+  name: 'mobilenetv3_large_21k'
+  type: 'multilabel'
+  pretrained: True
+  save_chkpt: True
+  feature_dim: 1280
+  dropout_cls:
+    p: 0.2
+
+custom_datasets:
+  roots: ['/data/train.json', '/data/val.json']
+  types: ['multilabel_classification', 'multilabel_classification']
+  names: ['train_data', 'val_data']
+
+data:
+  root: './'
+  sources: ['train_data']
+  targets: ['val_data']
+  height: 448
+  width: 448
+  norm_mean: [0.485, 0.456, 0.406]
+  norm_std: [0.229, 0.224, 0.225]
+  save_dir: 'output/mulitilabel/mobilenetv3'
+  workers: 6
+  transforms:
+    random_flip:
+      enable: True
+      p: 0.5
+    randaugment:
+      enable: True
+    cutout:
+      enable: True
+      cutout_factor: 0.3
+      p: 0.35
+
+loss:
+  name: 'asl'
+  softmax:
+    s: 1.0
+    compute_s: False
+  asl:
+    gamma_pos: 0.
+    gamma_neg: 4.
+
+sampler:
+  train_sampler: 'RandomSampler'
+
+metric_losses:
+  enable: False
+
+train:
+  optim: 'sam'
+  lr: 0.0002
+  nbd: True
+  max_epoch: 80
+  weight_decay: 5e-4
+  batch_size: 32
+  lr_scheduler: 'onecycle'
+  early_stoping: True
+  pct_start: 0.2
+  train_patience: 5
+  lr_decay_factor: 1e3
+  deterministic: True
+  target_metric: test_acc
+  gamma: 0.1
+  ema:
+    enable: True
+    ema_decay: 0.9997
+
+sam:
+  enable: True
```

Comment:
I think this flag introduces ambiguity, since SAM can be also specified via train.optim.
I'd propose remove this section and move rho the to train. or train.sam group.

Reply:
Agree. Ok